staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -417,6 +420,19 @@ type Filesystem struct {
 	Virtiofs *FilesystemVirtiofs `json:"virtiofs"`
 }
 
+//
+// +k8s:openapi-gen=true
+type Sound struct {
+	// Mandatory attribute, which specifies what real sound device is emulated.
+	// Supported values:  es1370, sb16, ac97, ich6, usb.
+	// More info: https://libvirt.org/formatdomain.html#sound-devices
+	Model string `json:"model"`
```

Comment:
We need api webhook validation for these fields. see `pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go`